### ISSUE TITLE
 shows this mesagge on Android auto and isn't working 

### ISSUE THREAD

**Issue by giokks** [slashmax/AAMirror#99]
![20210523_123953](https://user-images.githubusercontent.com/84662594/119255482-c2bbbe00-bbc4-11eb-9ad7-229aa16eca29.jpg)

**Comment by digboiyeah** [comment]
Yes, I tried the this today, and the same message appears, I think it's becuase it's an old version (2 years old).

**Comment by Lucky225** [comment]
You have to use AA Phenotype Patcher to patch AA Mirror, reboot, and then it should work. https://github.com/Eselter/AA-Phenotype-Patcher -- To the OP good luck, looks like you have a 2020-21 Ford Escape as well. It works for me but when you launch an app touch doesn't work from the infotainment to the app, which means I still have to use my phone which makes my use case useless :(

**Comment by Denusdv** [comment]
I had to upgrade the AA to 6.8 ... version otherwise the android doesn't allow to start it.
once it got updated I start getting "NO NEW MESSAGES DURING THIS DRIVE"
Looks like the patcher is intended to handle another issue, btw it was updated for three years.
Is there somebody who managed to fix "NO NEW MESSAGES DURING THIS DRIVE" problem

**Comment by Tiemichael** [comment]
AA Mirror was not updated for quite some time.
Why don't you try [AA AIO Tweaker](https://forum.xda-developers.com/t/root-only-aa-aio-tweaker-the-ultimate-android-auto-utility.4194239) 
I also had this "NO NEW MESSAGES DURING THIS DRIVE" problem - re-patching with AIO Tweaker helped

**Comment by Lucky225** [comment]
> You have to use AA Phenotype Patcher to patch AA Mirror, reboot, and then it should work. https://github.com/Eselter/AA-Phenotype-Patcher -- To the OP good luck, looks like you have a 2020-21 Ford Escape as well. It works for me but when you launch an app touch doesn't work from the infotainment to the app, which means I still have to use my phone which makes my use case useless :(

I should update my response, screen2auto fixes the touch not working, you have to launch it first then go back to aamirror and aamirror works like a charm :) 

![image](https://user-images.githubusercontent.com/3251120/130343979-e803e83c-3cb8-49d1-9657-c0ba7d8ec716.png)
![image](https://user-images.githubusercontent.com/3251120/130343996-5091f8bb-bf5a-4338-be00-525476ede112.png)

**Comment by Denusdv** [comment]
Could you tell me if I have to have a rooted device to apply AA AIO Tweaker
The reason I'm asking, in my case the device is not rooted when I open AA AIO Tweaker,
the following message shows up. 
.![Screenshot_20210822-104857_AA AIO Tweaker](https://user-images.githubusercontent.com/2059461/130346972-3cb39833-4e79-4ffc-8be7-1fde0d66d75e.jpg)

pressing on "OK" does nothing but closes the popup. 
I guess it has to open a screen with options

Any idea will be appreciated

**Comment by Tiemichael** [comment]
Not sure if it only works with root ...
Better check the XDA-thread, I am sure it's answered there

You can also check  https://github.com/Eselter/AA-Phenotype-Patcher.
